cura/Settings/ContainerManager.py

```diff
@@ -693,6 +696,70 @@ def duplicateMaterial(self, material_id: str) -> str:
             duplicated_container.deserialize(f.read())
         duplicated_container.setDirty(True)
         self._container_registry.addContainer(duplicated_container)
+        return new_id
+
+    ##  Create a new material by cloning Generic PLA and setting the GUID to something unqiue
+    #   \return \type{str} the id of the newly created container.
+    @pyqtSlot(result = str)
+    def createMaterial(self) -> str:
+        # Ensure all settings are saved.
+        Application.getInstance().saveSettings()
+
+        containers = self._container_registry.findInstanceContainers(id="generic_pla")
```

Comment:
Both Brand and Material Type are now set to "Custom", inviting the user to enter the proper value.